ophyd/ophydobj.py

```diff
@@ -72,6 +81,7 @@ def __init__(self, *, name=None, parent=None, labels=None,
             raise ValueError("name must be a string.")
         self._name = name
         self._parent = parent
+        self.est_time = est_time(self)
```

Comment:
Reminder to all of us to circle back to whether we can get away with `est_time(self.name)` or whether we really need to pass in the live object here. If `est_time` becomes responsible for generating the status objects, then we need the live object.

Reply:
I have made this change, as I only ever used self.name inside the est_time classes